### DISCUSSION TITLE
What happened to the "play" script button, any way to restore it or a shortcut?

### DISCUSSION THREAD

**Discussion by MorganeLeCaignec** [TriliumNext/Trilium#1766] [Q&A] [answered]
Since the latest update 0.46.5 my render notes have lost the "play/render script" button which was very useful as it allowed me to reset a complex html/js render without reloading the full UI.
Some of those render notes have promoted attributes that I often change and re-render the note to take into account the programmatic changes, so that button/feature was very useful.

**Comment by zadam** [comment, marked as answer]
Hi, so I readded this to the menu:

![image](https://user-images.githubusercontent.com/617641/111844481-678af800-8903-11eb-911c-ec7547b9b213.png)

you can also assign a shortcut to it in the options - the name is "renderActiveNote"
